pxtblocks/blocklycompiler.ts

```diff
@@ -1460,6 +1461,7 @@ namespace pxt.blocks {
                         isExpression: fn.retType && fn.retType !== "void",
                         imageLiteral: fn.attributes.imageLiteral,
                         hasHandler: !!comp.handlerArgs.length || fn.parameters && fn.parameters.some(p => (p.type == "() => void" || !!p.properties)),
+                        handlerKey: fn.attributes.blockHandlerKey,
```

Comment:
You're already passing all the attributes in this object; no need to change the interface